src/components/App/App.css

```diff
@@ -0,0 +1,8 @@
+.wrap {
+    margin: auto;
+    width: 300px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=5937957)